stubs/console-menu/consolemenu/console_menu.pyi

```diff
@@ -0,0 +1,86 @@
+from consolemenu.menu_formatter import MenuFormatBuilder as MenuFormatBuilder
+from consolemenu.screen import Screen as Screen
+
+class ConsoleMenu:
+    currently_active_menu: ConsoleMenu | None
+    screen: Screen
+    clear_screen_before_render: bool
+    formatter: MenuFormatBuilder
+    title: str | None
+    subtitle: str | None
+    prologue_text: str | None
+    epilogue_text: str | None
+    highlight: None
+    normal: None
+    show_exit_option: bool
+    items: list[MenuItem]
+    parent: ConsoleMenu
+    exit_item: ExitItem
+    current_option: int
+    selected_option: int
+    returned_value: object
+    should_exit: bool
+    previous_active_menu: ConsoleMenu
+    def __init__(
+        self,
+        title: str | None = ...,
```

Comment:
May also be a `Callable[[], str]`. This also goes for `subtitle`, `prologue_text`, `epilogue_text`. See the code for the `get_title` method.